IsIdentifiable/Whitelists/CsvWhitelist.cs

```diff
@@ -4,7 +4,7 @@
 using CsvHelper;
 using CsvHelper.Configuration;
 
-namespace IsIdentifiable.Allowlists;
+namespace IsIdentifiable.Whitelists;
```

Comment:
What's the reason for the change here? I think we should prefer allowlist over whitelist

Reply:
Agreed - the namespace inside the file said "allowlists" but the actual filename was still "whitelists" so the discrepancy got flagged.